internal/archive/archive.go

```diff
@@ -364,12 +364,12 @@ func (ta *tarAppender) addTarFile(path, name string) error {
 	//by the kernel and already have proper ownership relative to the host
 	if !isOverlayWhiteout &&
 		!strings.HasPrefix(filepath.Base(hdr.Name), WhiteoutPrefix) &&
-		!ta.IDMappings.Empty() {
-		fileIDPair, err := getFileUIDGID(fi.Sys())
+		!ta.IdentityMapping.Empty() {
+		fileIdentity, err := getFileUIDGID(fi.Sys())
```

Comment:
Should we rename this to getFileIdentity?

Reply:
Sounds good. Let's do it.